<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QWidget" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../77-770172_dollar-clipart-psd-bag-of-money-icon-png.jpeg</normaloff>../../../../77-770172_dollar-clipart-psd-bag-of-money-icon-png.jpeg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0,0,0,0,0" columnstretch="1,0,0,0,0,0,0,0,0,0,0,0">
   <item row="5" column="11">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="8">
    <widget class="QWidget" name="loginWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4,1">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>62</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="3,0,0,1,3,1,1,0,2">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="titleLabel">
          <property name="font">
           <font>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Spend &amp; Note</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="loginLabel">
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="loginLineEdit">
            <property name="placeholderText">
             <string>Login.....</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="passwordLabel">
            <property name="text">
             <string>Password</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="passwordLineEdit">
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Password.....</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="loginErrorLabel">
          <property name="styleSheet">
           <string notr="true">color: rgb(204, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loginSubmitButton">
          <property name="styleSheet">
           <string notr="true">color: rgb(243, 243, 243);
background-color: rgb(78, 154, 6);</string>
          </property>
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="signupLabel">
          <property name="text">
           <string>Do not have an account yet?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="signupButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(243, 243, 243);</string>
          </property>
          <property name="text">
           <string>Sign Up</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Language"/>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>62</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="8" rowspan="3" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>93</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="12">
    <widget class="QWidget" name="ErrorWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 41, 41);
color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,1,1,3,0">
        <item>
         <widget class="QToolButton" name="icon">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../resources.qrc">
            <normaloff>:/Icons/Icons/baseline_warning_black_24dp.png</normaloff>:/Icons/Icons/baseline_warning_black_24dp.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Login failed!</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ErrorTitleLabel">
          <property name="text">
           <string>Error code:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ErrorCodeLabel">
          <property name="text">
           <string>*code*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ErrorDescriptionLabel">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>*description*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="CloseErrorBannerToolButton">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="8">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>93</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="11">
    <widget class="QComboBox" name="LanguageSelector">
     <item>
      <property name="text">
       <string>English</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Українська</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>22</width>
       <height>2</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Icons/Icons/baseline_language_black_24dp.png) no-repeat;
background-size:contain;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
